hw/ip/i2c/dv/i2c_sim_cfg.hjson

```diff
@@ -53,10 +52,9 @@
 
   // List of regressions.
   regressions: [
-    // TODO: Add i2c_sanity test to the sanity suite once it is passing.
-    // {
-    //   name: sanity
-    //   tests: ["i2c_sanity"]
-    // }
+    {
+       name: sanity
+       tests: ["i2c_sanity"]
```

Comment:
is i2c_sanity passing? if not, we can't enable it here, as it will break the CI

Reply:
Ops, fixed! I need to split a big PR into several small ones for easy reviewing. Therefore, some stuff should be commented out in intermediate PR to pass CI then enable in the final PR.